### ISSUE TITLE
Allow Sidenav to peek out by not closing entirely

### ISSUE THREAD

**Issue by Dogfalo** [Dogfalo/materialize#2304]
See #2277

**Comment by vincentmorneau** [comment]
Any plans to implement this one?

**Comment by tomscholz** [comment]
Is [this](https://github.com/Dogfalo/materialize/issues/3692) related?

**Comment by vincentmorneau** [comment]
Yes it's very similar. No need to have duplicate issues, but since the other one is closed I would keep this one alive. It's a pretty neat feature.

**Comment by srinivasrk** [comment]
Is this feature still on track ? Is there a open PR ? If not I would like to take a stab at it. If you could point me in the right direction
